unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1140,6 +1134,19 @@ loop e = do
               Cli.respond $ SlurpOutput input (PPE.suffixifiedPPE ppe) sr
               addDefaultMetadata adds
               syncRoot description
+            SaveExecuteResultI resultName -> do
+              description <- inputDescription input
+              let resultVar = Name.toVar resultName
```

Comment:
What happens if this is an absolute name?
I did a quick look and I think it'd just add a var with a leading dot and I'm not sure we handle that correctly right now (though for this use-case it'd be nice if we did), alternatively we could bail out on an Absolute name.

I haven't read the transcripts yet, but it'd be good to have a transcript test for this 👍🏼